atlasdb-impl-shared/src/main/metrics/targeted-sweep.yml

```diff
@@ -36,6 +36,14 @@ namespaces:
             2. WAITING_UNTIL_CLOSEABLE
             3. CLOSING_FROM_OPEN
             4. IMMEDIATELY_CLOSING
+      estimatedPendingNumberOfBucketsToBeSwept:
```

Comment:
Yep, what you've explained in the paragraph below is correct, though I struggle to think of a good, succinct name for this without making people think "oh, this is the number of pending buckets for the whole queue" and/or think it's some proxy for the age of the oldest queue entries.

Reply:
No action needed right now.